/**:
    ros__parameters:
      video_device: "/jetcocam"
      framerate: 30.0
      io_method: "mmap"
      frame_id: "jetcocam"
      pixel_format: "yuyv"  # see usb_cam/supported>
      av_device_format: "YUV422P"
      image_width: 640
      image_height: 480
      camera_name: "test_camera"
      # camera_info_url: "package://usb_cam/config/ca>
      camera_info_url: "package://jetcobot_bringup/config/jetcocam_1_calib.yaml"
      brightness: -1
      contrast: -1
      saturation: -1
      sharpness: -1
      gain: -1
      auto_white_balance: true
      white_balance: 4000
      autoexposure: true
      exposure: 100
      autofocus: false
      focus: -1